### PR TITLE
Add getters for rrules, exrules, rdates, exdates

### PR DESCRIPTION
**Pull request by epicfaace**
---
- Add getters for rrules, exrules, rdates, exdates. That way we have a public API to accomplish the issue that https://github.com/jakubroztocil/rrule/issues/212 is describing.

### Thanks for contributing to `rrule`!

To submit a pull request, please verify that you have done the following:

- [x] Merged in or rebased on the latest `master` commit
- [x] Linked to an existing bug or issue describing the bug or feature you're
      addressing
- [x] Written one or more tests showing that your change works as advertised
- [ ] Run `yarn build` to rebuild the `dist/` files

### CODE REVIEW

**Comment by epicfaace** [comment]
@davidgoli sure thing -- I agree with your logic, our use case is not to modify them at all -- we just need to be able to generate RRules from an RRuleset for an RRuleset visual editor. I've modified the PR accordingly.

File: test/rruleset.test.ts [hunk 1]
```diff
@@ -727,4 +727,45 @@ describe('RRuleSet', function () {
...
+      let rrule = new RRule({
+        freq: RRule.YEARLY,
+        count: 2,
+        dtstart: parse('19600101T090000'),
+        tzid: 'America/New_York'
+      });
+      set.rrule(rrule);
+
+      expect(set.rrules().map(e => e.toString())).eql([rrule.toString()]);
+    });
```
Comment (davidgoli) [inline test/rruleset.test.ts]: one small style nitpick: can we make sure there are blank lines in between `it` blocks? thanks!

Comment (epicfaace) [reply test/rruleset.test.ts]: Done

**Comment by epicfaace** [comment]
@davidgoli could you create a release for this? Would be greatly appreciated -- thanks!

**Comment by davidgoli** [comment]
@epicfaace Published v2.6.2. Cheers!

**Pull request merged**